cni/pkg/repair/options.go

```diff
@@ -0,0 +1,149 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package repair
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/pflag"
+	"github.com/spf13/viper"
+
+	"istio.io/istio/tools/istio-iptables/pkg/constants"
+	"istio.io/pkg/log"
+)
+
+type controllerOptions struct {
+	RepairOptions *Options `json:"repair_options"`
+	RunAsDaemon   bool     `json:"run_as_daemon"`
+	Enabled       bool     `json:"enabled"`
+}
+
+func init() {
```

Comment:
Is this all net new code including options?

Reply:
They were copied from the original repair main.go.